linter.py

```diff
@@ -1,57 +1,72 @@
 #
 # linter.py
-# Linter for SublimeLinter3, a code checking framework for Sublime Text 3
+# Linter for SublimeLinter4, a code checking framework for Sublime Text 3
 #
 # Written by Noah Davis
 # Copyright (c) 2016 Noah Davis
 #
+# Updated by Schuyler Jager & Andreas for SublimeLinter 4
+# Copyright (c) 2020 Schuyler Jager & Andreas
 # License: MIT
 #
 
 """This module exports the Codeclimate plugin class."""
+import logging
+import os
+from SublimeLinter.lint import Linter, util
 
-from SublimeLinter.lint import Linter, util, persist
+
+logger = logging.getLogger('SublimeLinter.plugin.eslint')
```

Comment:
`eslint` -> `codeclimate` 

*eslint* is a different linter.

Reply:
I removed the logger messages for now.